src/Microsoft.ML.Transforms/Text/NgramTransform.cs

```diff
@@ -679,12 +696,16 @@ public sealed class NgramExtractingEstimator : IEstimator<NgramExtractingTransfo
         /// </summary>
         public enum WeightingCriteria
         {
+            /// <summary>Term Frequency. Calculated based on the number of occurrences in the document.</summary>
             [EnumValueDisplay("TF (Term Frequency)")]
             Tf = 0,
 
+            /// <summary>Inverse Document Frequency. A ratio (the logarithm of inverse relative frequency)
```

Comment:
>/// <summary [](start = 12, length = 12)

thank you for adding this.
Should we make it 
```
<summary>
text
</summary>
```
?

Reply:
Yes you are right!

---
In reply to: [276869289](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276869289) [](ancestors = 276869289)